src/App.css

```diff
@@ -11,8 +11,8 @@
 
 .card {
   display: inline-grid;
-  grid-template-columns: 12vw;
-  grid-template-rows: 12vh;
+  grid-template-columns: 125px;
```

Comment:
When viewed in the browser, although you have made it responsive, your grids seem to overlap a little. Take a look at this and change the styling 👍